spacy/errors.py

```diff
@@ -26,7 +27,17 @@ def setup_default_warnings():
     filter_warning("once", error_msg="[W114]")
 
 
-def filter_warning(action: str, error_msg: str):
+def filter_warning(
+    action: Union[
+        Literal["default"],
+        Literal["error"],
+        Literal["ignore"],
+        Literal["always"],
+        Literal["module"],
+        Literal["once"],
+    ],
```

Comment:
Could this be specified more simply by combining the Literal values?
```suggestion
    action: Literal["default", "error", "ignore", "always", "module", "once"],
```

Reply:
So much has changed in Thinc since I started this PR that I've decided to close it and start again. However, thanks for this suggestion which I will definitely adopt in the new PR.